### PR TITLE
Support for native services

### LINKED ISSUE #2243: Provide a native service option
There is no option to include a native service (com.package.myservice) to AndroidManifest.xml, which prevents those Java services from being executed.

A CLI option --native-service, similar to --service, could allow us to specify the extra service(s) if any.

### PR DESCRIPTION
**Pull request by lerela**
Fixes #2243

### CODE REVIEW

File: pythonforandroid/bootstraps/common/build/build.py [hunk 1]
```diff
@@ -619,6 +620,9 @@ def parse_args_and_make_package(args=None):
     ap.add_argument('--service', dest='services', action='append', default=[],
                     help='Declare a new service entrypoint: '
                          'NAME:PATH_TO_PY[:foreground]')
+    ap.add_argument('--native-service', dest='native_services', action='append', default=[],
+                    help='Declare a new native service: '
+                         'package.name.service')
```
Comment (AndreMiras) [inline pythonforandroid/bootstraps/common/build/build.py:625]: Minor: no sure why this line is broken down in two

Comment (lerela) [reply pythonforandroid/bootstraps/common/build/build.py:625]: No specific reason, I just used the same codestyle than for the previous argument (line 620/621) for consistency.

Thanks for the merge!

**Comment by obfusk** [comment]
Was it intentional to merge this into `master` instead of `develop`?

**Comment by tshirtman** [comment]
ow, probably an overlook!

**Comment by lerela** [comment]
Ah, that's my mistake, GitHub didn't pick the default branch when I created the PR therefore it targeted `master`. Sorry about that.

**Comment by AndreMiras** [comment]
ouch I completely missed that :facepalm:

**Pull request merged**